Context improvements

Vinai opened the pull request:
Added several improvements:
- add $this->locatePath() to iAmOn() step
- add iLogInAsCustomerWithPassword step
- add additional step regex to iLoginAsAdmin
- add public getter for sessionService
- add customerLogin and customerLogout service methods

Vinai committed:
Add customerLogin and customerLogout service methods

Vinai committed:
Add public getter for sessionService

Vinai committed:
Add additional step regex to iLoginAsAdmin

Vinai committed:
Add iLogInAsCustomerWithPassword step

Vinai committed:
Add $this->locatePath() to iAmOn() step

alistairstead commented on src/MageTest/MagentoExtension/Service/Session.php:
```diff
@@ -81,4 +81,40 @@ public function adminLogin($username, $password)
+            $_SESSION = null;
+        }
+
+        // I once locked my mother in the house and then walked against a wall and
```
:guardsman: Should this line be here?

Vinai replied:
Um, I wondered where that went - obviously not into the skype chat window :scream:

Vinai committed:
Update Session.php

Remove embarrassing part of a story that was meant to be typed into another window :)
